bindings/nodejs/lib/types/block/input/index.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+export * from './input';
```

Comment:
Shouldn't the content of `./input` be just in this file here or is it common to still re export a single file?